experiments/ClimaEarth/user_io/postprocessing.jl

```diff
@@ -40,6 +41,7 @@ function postprocess_sim(cs, postprocessing_vars)
             leaderboard_base_path = artifact_dir
             compute_leaderboard(leaderboard_base_path, atmos_output_dir, 3)
             compute_pfull_leaderboard(leaderboard_base_path, atmos_output_dir, 6)
+            test_rmse_thresholds(atmos_output_dir, 3)
```

Comment:
does this function also require `pressure_in_output`, or can it be outside of that check?